pkg/controllers/provisioning/v1alpha1/allocation/suite_test.go

```diff
@@ -77,9 +78,11 @@ var _ = Describe("Allocation", func() {
 				Cluster: &v1alpha1.ClusterSpec{Name: "test-cluster", Endpoint: "http://test-cluster", CABundle: "dGVzdC1jbHVzdGVyCg=="},
 			},
 		}
+		ctx = context.Background()
```

Comment:
I know it might feel ugly, but I might just define these contexts inline. Up to you, but it's fewer lines of code.

Reply:
I personally would prefer the two extra lines vs the ugliness of repeating context.Background() lol